apps/docs/content/guides/platform/metrics.mdx

```diff
@@ -48,4 +48,247 @@ scrape_configs:
+                ```text
+                docker compose up
+                ```
+            </StepHikeCompact.Code>
+
+    </StepHikeCompact.Step>
+
+    <StepHikeCompact.Step step={6}>
+        <StepHikeCompact.Details title="Login" >
+            Login with your username (default is `admin`) and the password you created in step 4.
+        </StepHikeCompact.Details>
+            <StepHikeCompact.Code>
+                ![Grafana login](/docs/img/guides/platform/grafana-login.png)
+            </StepHikeCompact.Code>
+    </StepHikeCompact.Step>
+
+</StepHikeCompact>
+
+### Deploying to Grafana cloud
+
+<StepHikeCompact>
+
+    <StepHikeCompact.Step step={1}>
+
+        <StepHikeCompact.Details title="Create a free Grafana Cloud account" >
+        Go to Grafana\'s [registration form](https://grafana.com/auth/sign-up/create-user?pg=login) and create an account
+        </StepHikeCompact.Details>
+            <StepHikeCompact.Code>
+                ![Grafana Registration Page](/docs/img/guides/platform/grafana-registration.png)
+            </StepHikeCompact.Code>
+    </StepHikeCompact.Step>
+
+
+    <StepHikeCompact.Step step={2} >
+
+    <StepHikeCompact.Details title="Give a name to your Grafana project">
+    Grafana will prompt you to create a stack. Give it a name and continue.
+    </StepHikeCompact.Details>
+    <StepHikeCompact.Code>
+    ![Create Grafana Stack](/docs/img/guides/platform/create-grafana-stack.png)
+    </StepHikeCompact.Code>
+
+    </StepHikeCompact.Step>
+
+    <StepHikeCompact.Step step={3} >
+
+    <StepHikeCompact.Details title="Create a new Grafana Dashboard">
+        Create a new Dashboard
+    </StepHikeCompact.Details>
+    <StepHikeCompact.Code>
+        ![Select Grafana Dashboard](/docs/img/guides/platform/select-grafana-dashboard.png)
+    </StepHikeCompact.Code>
+
+
+    </StepHikeCompact.Step>
+    <StepHikeCompact.Step step={4} >
+    <StepHikeCompact.Details title="Search for the Supabase template" />
+
+        <StepHikeCompact.Code>
+            ![Select Grafana Dashboard](/docs/img/guides/platform/supabase-grafana-template.png)
+        </StepHikeCompact.Code>
+
+
+    </StepHikeCompact.Step>
+
+        <StepHikeCompact.Step step={5} >
+    <StepHikeCompact.Details title="Add your credentials and deploy">
+        Add your service_role key from the [API Settings](https://supabase.com/dashboard/project/_/settings/api) and Reference ID from the [General Settings](https://supabase.com/dashboard/project/_/settings/general) to your Grafana credentials. Then install your dashboard.
+    </StepHikeCompact.Details>
+    <StepHikeCompact.Code>
+        ![Add credentials](/docs/img/guides/platform/add-credentials-grafana.png)
+    </StepHikeCompact.Code>
+
+    </StepHikeCompact.Step>
+
+</StepHikeCompact>
+
+### Deploying to fly.io
+
+<Admonition type="note">
+  This section explains deploying to Fly.io using the included configuration file, but deployment to
+  other hosts is also possible.
+</Admonition>
```

Comment:
```suggestion
<Admonition type="note">

This section explains deploying to Fly.io using the included configuration file, but deployment to other hosts is also possible.

</Admonition>
```

Add empty lines to prevent Prettier inserting line breaks awkwardly